source/manual/alerts/content-performance-manager-app-healthcheck-not-ok.html.md

```diff
@@ -12,11 +12,20 @@ Please notify Data Informed Content via Slack `#govuk-data-informed` about the a
 
 If there is a health check error showing for Content Performance Manager, you can click on the alert to find out more details about what’s wrong. Here are the possible problems you may see:
 
+## ETL :: no aggregations of metrics for yesterday
+
+This means that [the ETL master process][1] that runs daily that creates aggregations of the metrics failed.
```

Comment:
Could you please be specific about the monthly aggregations (also in the title of the error message)?

Reply:
Thanks @pmanrubia - [fixed](https://github.com/alphagov/govuk-developer-docs/commit/815cf09d5697482bae97e3523680e6904c35a62f)